app/models/tag.rb

```diff
@@ -61,14 +64,34 @@ def default_tagables_for_homepage(tagable_category, page = 1)
   end
 
   def entities_for_homepage(page = 1)
+    %w[Person Org].reduce({}) do |acc, type|
+      acc.merge(type => paginate(page, *sort_and_count_entities(type, page)))
+    end
+  end
+
+  def paginate(page, records, count)
```

Comment:
cool! 

we could also make it extra generic like such:

``` rb
def paginate(page, records, count, per)
  Kaminari
        .paginate_array(records, total_count: count)
        .page(page)
        .per(per)
end
```

and put it on ApplicationController if we think other controllers might want to use it.

Reply:
+1, though i'd switch the argument order to `page, per, records, count` so you can splat records and count (as we do here when we return a `[records, count]` tuple-as-array)

Reply:
@aepyornis -- `#paginate` is on model code here, though it would likely most often get used in controller code... is there someplace we could extract it that would be available in both models and controllers?

Reply:
Ah right. Well, to share things between models this app generally uses concerns....perhaps it can go with the tagable concern? It can also just say where it is of course...

Reply:
it seems like something we might want to do often enough (paginate an arbitrary array) that it's worth extracting. and the applicable use case would be anything (not just tagables...) i just don't know where the convention is to extract it...